draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -201,21 +199,16 @@ multiple paths. In addition, the multipath extensions enable to tradeoff timelin
 which is important for low-latency applications that do not require
 guaranteed delivery services, such as Audio/Video streaming.
 
-MP-DCCP was first suggested
-in the context of the 3GPP work on 5G multi-access solutions
-{{I-D.amend-tsvwg-multipath-framework-mpdccp}} and for hybrid access
-networks {{I-D.lhwxz-hybrid-access-network-architecture}}{{I-D.muley-network-based-bonding-hybrid-access}}, where
-MP-DCCP can be applied for load-balancing, seamless session handover, and bandwidth
-aggregation (referred to as Access Traffic Steering, Switching, and Splitting (ATSSS)
-in the 3GPP terminology {{TS23.501}}).
+In addition to the integration into DCCP services, other use cases of MP-DCCP are the support of
+3GPP 5G multi-access solutions {{I-D.amend-tsvwg-multipath-framework-mpdccp}} or hybrid access networks
+{{I-D.lhwxz-hybrid-access-network-architecture}}{{I-D.muley-network-based-bonding-hybrid-access}}, in
+which MP-DCCP can be used for load balancing, seamless session handover and bandwidth aggregation
+(referred to as Access Traffic Steering, Switching, and Splitting (ATSSS) in 3GPP terminology {{TS23.501}}).
```

Comment:
I'm afraid this may be misinterpreted as if MP-DCCP is part of the ATSSS spec, which is not.

It was OK to have this text in early versions of the spec, but I think we can get rid.

Reply:
Med,

I would like to keep a reference to these possible use cases to give readers some orientation.
Since I agree with you that the current text leaves room for misunderstanding, I have changed it to https://github.com/tsvwg/ietf-multipath-dccp/pull/295/commits/d64e6ef72fc925066bb4fd064cac7d2d48654d98. Is this acceptable now?